thoth-export-server/src/xml/onix3_thoth.rs

```diff
@@ -1150,7 +1150,12 @@ impl XmlElementBlock<Onix3Thoth> for WorkIssues {
                 })?;
                 write_element_block("IDValue", w, |w| {
                     w.write(XmlEvent::Characters(
```

Comment:
this element should only be output if there is a value (like with `series_url`, below)